tools/Customization/DevHome.Customization/Views/DevDriveInsightsView.xaml

```diff
@@ -103,7 +103,7 @@
             <TextBlock
                 Margin="0 20 0 -10"
                 Style="{StaticResource CardBodyStackPanelTextBlockStyle}"
-                Text="Dev Drive Volumes"
+                Text="{x:Bind ViewModel.DevDriveVolumesText, Mode=OneWay}"

```

Comment:
If this doesn't change, don't make it a Property on the ViewModel, reference it directly here.
```suggestion
                x:Uid="DevDriveVolumesHeader.Text"
```
```
  <data name="DevDriveVolumesHeader.Text" xml:space="preserve">
    <value>Dev Drive volumes</value>
    <comment>Header for Dev Drive volumes list in Dev Drive insights page</comment>
  </data>
```